### DISCUSSION TITLE
Beta tag in Sidebar

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3847] [Help: Figma] [answered]
Hey team! 

I noticed the right side of the beta badge is getting cut off in the sidebar - not a huge deal, just wanted to let you know!

![Screenshot 2024-04-03 at 2 54 10 PM](https://github.com/twilio-labs/paste/assets/87141551/d351c72b-e8d5-47e8-9f57-6842f3eb454a)

Thanks!

**Comment by serifluous** [comment]
Hey @ceyerkes ! Where is this? In prod or Figma?

**Reply by ceyerkes** [reply]
I think just Figma!

**Reply by serifluous** [reply, marked as answer]
Just published the fix in Figma. Thanks for catching this!
